mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule UeberauthFacebook.Mixfile do
 
   defp deps do
     [{:ueberauth, "~> 0.4"},
-     {:oauth2, "0.6.0"},
+     {:oauth2, "~> 0.8.0"},
```

Comment:
I think to be consistent with other strategies, this would be `~> 0.8`.

see: https://github.com/ueberauth/ueberauth_google/pull/25/files